Sources/Plasma/PubUtilLib/plModifier/plImageLibMod.h

```diff
@@ -75,6 +75,8 @@ class plImageLibMod : public plSingleModifier
     };
 
     size_t  GetNumImages() const { return fImages.size(); }
+    plBitmap* GetImage(const ST::string&) const;
+    const std::vector<ST::string> GetImageNames(void) const;
```

Comment:
```suggestion
    const std::vector<ST::string> GetImageNames() const;
```

Reply:
I noticed I had these and thought I nuked them all.  Guess I missed one. ;)